app/containers/Applications/ApplicationWizardStep.tsx

```diff
@@ -50,6 +50,7 @@ const ApplicationWizardStep: React.FunctionComponent<Props> = ({
 
   const onValueChanged = async change => {
     const {formData} = change;
+    console.log(formData);
```

Comment:
how would you feel about turning on [no-console](https://eslint.org/docs/rules/no-console)

Reply:
@dleard may object to that (there's a lot of mutation responses being logged in the console right now)